New-PSPowerHour.ps1

```diff
@@ -0,0 +1,34 @@
+<#
+    .SYNOPSIS
+        Generate the markdown table of the presenters for the PSPowerHour
+    .NOTES
+        This utilizes PowerShellForGitHub module by the PowerShell team
+        For best results and the way written you will need a token from GitHub (no permissions needed to be assigned)
+#>
+[cmdletbinding()]
+param(
+    [string]$GitHubToken
+)
+$global:gitHubApiToken
+foreach ($m in 'PowerShellForGitHub', 'PSMarkdown') {
+    if (-not (Get-Module $m -ListAvailable)) {
+        throw "Please install $m module"
+    }
+    else {
+        Import-Module $m
+    }
+}
+
+$data = Get-GitHubIssueForRepository -repositoryUrl 'https://github.com/pspowerhour/pspowerhour' -gitHubAccessToken $GitHubToken |
+    Where-Object {$_.labels.name -eq 'proposal'} | Select-Object Number, Title, user | sort-object Number | select-object -first 10
+[int]$sequence = 1
+$data | ForEach-Object {
+    [PSCustomObject][ordered]@{
```

Comment:
You only need `[pscustomobject]` here, this is ordered by default

Reply:
It actually isn't, that is why I tried adding it but it does not work. When you pass it to the convert it changes the property order.

Reply:
Ahhh, just took a look... going to poke around other markdown modules, there are a few issues with PSMarkdown